CMakeLists.txt

```diff
@@ -43,7 +43,7 @@ if (NOT THRUST_TOPLEVEL_PROJECT)
   return()
 endif()
 
-option(THRUST_ENABLE_HEADER_TESTING "Test that all public headers compile." "ON")
+option(THRUST_ENABLE_HEADER_TESTING "Test that all public headers compile." ON)
```

Comment:
Both forms are valid, but why change one and not the others?

Reply:
I don't think I did this intentionally, it was just something I copied pasted.

Reply:
Fixed.